cirq/linalg/diagonalize_test.py

```diff
@@ -68,53 +68,38 @@ def random_bi_diagonalizable_pair(
     b = cirq.dot(u, z, v)
     return a, b
 
+def _get_assert_diagonalized_by_str(m, p, d):
+    # coverage: ignore
+    return 'm.round(3) : %f, p.round(3) : %f, ' \
```

Comment:
Prefer `.format(...)` to `%`, just to be consistent with the rest of the code base.